src/etl/internal/Adapter/repository.py

```
@@ -0,0 +1,71 @@
+        self._collection_name = collection_name
+
+        # Модель скачается автоматически при первом запуске
+        print(TextEmbedding.list_supported_models())
+        print(TextEmbedding.list_supported_models())
+        self._model = TextEmbedding(model_name="sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2")
+
+    async def save_batch(self, messages: List[MessageMetadata]) -> None:
+        try:
+            texts = [m.text if m.text else "" for m in messages]
+
+            # Генерируем векторы (FastEmbed работает со списками строк)
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)